less/v1/advancement.less

```diff
@@ -43,46 +43,71 @@
   }
 
   /* ----------------------------------------- */
-  /*  Two Column Configurations                */
+  /*  Column Configurations                    */
   /* ----------------------------------------- */
-  &.two-column {
-    --grid-two-column-left-size: 1fr;
-    --grid-two-column-right-size: 1fr;
-
+  &.two-column, &.three-column {
     form {
       display: grid;
-      grid-template-columns: var(--grid-two-column-left-size) var(--grid-two-column-right-size);
-      grid-template-areas: "left right";
-      grid-gap: 0.4em;
+      gap: 0.4em;
 
-      .left-column {
+      > .left-column {
         grid-area: left;
         display: flex;
         flex-direction: column;
 
         > .form-group { flex: none; }
         > .drop-target { flex: 1; }
       }
-      .right-column {
+      > .right-column {
         grid-area: right;
-
-        &.level-list {
-          label {
-            flex: 0.5;
-            padding-right: 0.5rem;
-            text-align: end;
-          }
-          :is(input[type="text"], input[type="number"])::placeholder {
-            opacity: 0.5;
-          }
+      }
+      > .level-list {
+        label {
+          flex: 0.5;
+          padding-right: 0.5rem;
+          text-align: end;
```

Comment:
<img width="803" alt="Screenshot 2024-04-30 at 15 30 09" src="https://github.com/foundryvtt/dnd5e/assets/19979839/cbe76127-7db5-464a-9710-8c8e1dbb4aae">

How about this, using an icon (with tooltip) instead of the whole "Replacement" text.

Reply:
I also have a thought for "Select All" that might be a bit nicer, but it might be worth a second PR.